modules/home/README.md

```
@@ -0,0 +1,181 @@
+- `dust` - duコマンドの可視化版
+- `duf` - dfコマンドの改良版
+- `procs` - psコマンドの代替
+- `bottom` - topコマンドの改良版
+- `delta` - Git diffの可視化
+- `lazygit` - Git TUI
+- `yazi` - ターミナルファイルマネージャー
+- `tmux` - ターミナルマルチプレクサー
+- その他多数
+
+### dev.nix
+**基本的な開発ツール**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)